app/views/home/index.html.slim

```diff
@@ -1 +1,23 @@
-h1 New Vasjuky Project
+  .jumbotron.baner
+    h1 Hello, Франківськ!
+    p.light До вашої уваги соцаліний проект під кодовою назвою "NewVasjuky" групи IF-070.WebUI-RoR,
+            який має на меті #{'#{ довга мотиваційна стрінга }'}...
+
+    .button.btn.btn-primary.btn-lg.center
+        = link_to 'Повідомити про проблему', new_issue_path
+  ul.nav.nav-pills role="tablist"
+    li role="presentation"
+        span.badge #{@count}
+        span.phrase Проблем додано
+    li role="presentation"
+        span.badge #{@approved_count}
+        span.phrase Проблем вирішено
+  .row
+    h3.indent Останні додані:
+    - @issues_feed.each do |issue|
+      .col-md-3
+        .thumbnail= link_to issues_url

```

Comment:
wrong html.
you cannot put "p" inside link

ALSO what is this

`.img style="background-image:url(#{issue.attachment})"`

no inline styles.  Use image_tag